lgc/patch/VertexFetch.cpp

```diff
@@ -586,26 +590,30 @@ bool LowerVertexFetch::runImpl(Module &module, PipelineState *pipelineState) {
   if (!pipelineState->isUnlinked() || !pipelineState->getVertexInputDescriptions().empty()) {
     // Whole-pipeline compilation (or shader compilation where we were given the vertex input descriptions).
     // Lower each vertex fetch.
-    for (CallInst *call : vertexFetches) {
+    for (InputImportGenericOp *import : vertexFetches) {
```

Comment:
Maybe name it to `vertexFetch` or `fetch`

Reply:
Renaming to `fetch` since `vertexFetch` is already used for the `VertexFetch` object.